CHANGELOG.md

```diff
@@ -43,6 +43,10 @@ Usage:
 * Fix #1325: `jkube.enricher.jkube-name.name` doesn't modify `.metadata.name` for generated manifests
 * Fix #1336: Add documentation and quickstart for using custom generators with Gradle Plugins
 * Fix #1362: VolumePermissionEnricher : Replace iteration with bulk Collection.addAll call
+<<<<<<< HEAD
+=======
```

Comment:
This is not good, plz remove the "merge lines"

Reply:
oh no- missed that, updated the changes